lms/static/sass/features/_course-experience.scss

```diff
@@ -445,11 +404,11 @@ button.accordion-trigger, button.prerequisite-button {
 }
 
 .fa-check.fa.complete-checkmark {
-  border: 1px solid $green;
+  border: 1px solid $green !important;
```

Comment:
These important flags are to avoid conflicting with the `fa {color: blue;}` bit above and because the selector is so incredibly specific that we want the final word. And just to make sure that fg and bg change together.

Reply:
i would try to shy away from using !important.  can start to make styling cumbersome down the line.

Reply:
OK, you're right. Now avoided by moving some CSS around.